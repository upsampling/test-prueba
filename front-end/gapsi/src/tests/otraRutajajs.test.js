import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';

import { OtraRutajajs } from '../pages/OtraRutajajs/OtraRutajajs';

describe('Pruebas en <OtraRutajajs />', ()=>{

    test('Se debe de mostrar <OtraRutajajs /> correctamente', () => {

        const wrapper = shallow( <OtraRutajajs />);
        expect( wrapper ).toMatchSnapshot(); 
    })

})





