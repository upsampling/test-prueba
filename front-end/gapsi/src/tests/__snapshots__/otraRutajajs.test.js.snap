// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <OtraRutajajs /> Se debe de mostrar <OtraRutajajs /> correctamente 1`] = `
<div
  id="container-main"
>
  <NavBar />
  <div
    id="sub-container"
  >
    <Card>
      <CardContent>
        <CardHeader
          className="card-title"
          textAlign="center"
        >
          Bienvenido a la otra página que se creó.
        </CardHeader>
        <CardMeta>
          <Segment>
            <Dimmer
              active={true}
            >
              <Loader>
                Cargando...
              </Loader>
            </Dimmer>
            <div
              className="card-content"
            >
              <img
                alt=""
                className="img-card"
                src="user2.png"
              />
              <p
                className="label-card"
              />
              <span
                className="date label-card"
              >
                Version : 
              </span>
            </div>
          </Segment>
        </CardMeta>
        <CardDescription>
          <Button
            as="button"
            content="Ingresa..."
            onClick={[Function]}
          />
        </CardDescription>
      </CardContent>
    </Card>
  </div>
</div>
`;
